### PR TITLE
Update public roadmap for Q4, January 2022

### PR DESCRIPTION
**Pull request by karlchillmaid**
This PR updates the roadmap in line with our goals for the next 3 months.

It also updates the summary of recent improvements.

### CODE REVIEW

File: app/templates/views/roadmap.html [hunk 1]
```diff
@@ -26,28 +26,29 @@ <h2 class="heading-medium">Things we’re working on</h2>
...
-    <li>Meet our targets for <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('main.accessibility_statement')}}">accessibility</a></li>
-    <li>Explore how we can make it easier for new teams to <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('main.get_started')}}">start using Notify</a></li>
-    <li>Explore ways to improve the quarterly <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('main.how_to_pay')}}">billing and invoicing</a> process</li>
-    <li>Help teams understand how their organisation uses Notify</li>    
+  <ul class="list list-bullet"> 
+    <li>Let services pick their own <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('main.branding_and_customisation')}}">branding for emails</a></li>
+    <li>Test our current capacity limits and rate limits</li>
+    <li>Carry out our annual IT health check</li>
+    <li>Make it easier for new teams to <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('main.get_started')}}">start using Notify</a></li>
+    <li>Help services to get better value for money</li>
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:34]:
```suggestion
  <ul class="list list-bullet">
   
    <li>Let services pick their own <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('main.branding_and_customisation')}}">branding for emails</a></li>
    <li>Test our current capacity limits and rate limits</li>
    <li>Carry out our annual IT health check</li>
    <li>Make it easier for new teams to <a class="govuk-link govuk-link--no-visited-state" href="{{url_for('main.get_started')}}">start using Notify</a></li>
    <li>Help services to get better value for money</li>
```

File: app/templates/views/roadmap.html [hunk 2]
```diff
@@ -27,34 +27,46 @@ <h2 class="heading-medium">Things we’re working on</h2>
...
+    <li>Explore ways to improve our daily letter printing capability</li>
 
   </ul>
 
   <h2 class="heading-medium" id="things-we-have-done">Things we’ve done</h2>
 
-  <p class="govuk-body">Since January 2021 we’ve sent 1 billion messages for services that were set up in response to the coronavirus (COVID-19) pandemic. This includes 4.5 million <a class="govuk-link govuk-link--no-visited-state" href="https://www.nhs.uk/conditions/coronavirus-covid-19/coronavirus-vaccination/book-coronavirus-vaccination/">vaccination</a> invitation letters and 700 million <a class="govuk-link govuk-link--no-visited-state" href="https://www.gov.uk/get-coronavirus-test">test results</a>.</p>
+  <p class="govuk-body">Since January 2021 we’ve sent X billion messages for coronavirus (COVID-19) services. This includes X million <a class="govuk-link govuk-link--no-visited-state" href="https://www.nhs.uk/conditions/coronavirus-covid-19/coronavirus-vaccination/coronavirus-booster-vaccine/">booster vaccination</a> invitations and X million <a class="govuk-link govuk-link--no-visited-state" href="https://www.gov.uk/get-coronavirus-test">test results</a>.</p>
+
+  <p class="govuk-body">We’ve also:</p>
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:60]:
```suggestion
  <p class="govuk-body">More recently we have:</p>
```

File: app/templates/views/roadmap.html [hunk 3]
```diff
@@ -27,34 +27,46 @@ <h2 class="heading-medium">Things we’re working on</h2>
...
+  <p class="govuk-body">Since January 2021 we’ve sent X billion messages for coronavirus (COVID-19) services. This includes X million <a class="govuk-link govuk-link--no-visited-state" href="https://www.nhs.uk/conditions/coronavirus-covid-19/coronavirus-vaccination/coronavirus-booster-vaccine/">booster vaccination</a> invitations and X million <a class="govuk-link govuk-link--no-visited-state" href="https://www.gov.uk/get-coronavirus-test">test results</a>.</p>
+
+  <p class="govuk-body">More recently we have:</p>
+
+  <ul class="list list-bullet">
+
+    <li>improved our performance by upgrading the software we use</li>
+    <li>added more useful information to our quarterly billing reports</li>
+    <li>increased our capacity at short notice to meet a spike in demand</li>
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:66]:
```suggestion
    <li>improved our performance by upgrading the software we use</li>
    <li>increased our capacity at short notice to meet a spike in demand</li>
    <li>made quarterly billing reports available for organisations to download</li>

```

File: app/templates/views/roadmap.html [hunk 4]
```diff
@@ -27,34 +27,46 @@ <h2 class="heading-medium">Things we’re working on</h2>
...
+    <li>Explore ways to improve our daily letter printing capability</li>
 
   </ul>
 
   <h2 class="heading-medium" id="things-we-have-done">Things we’ve done</h2>
 
-  <p class="govuk-body">Since January 2021 we’ve sent 1 billion messages for services that were set up in response to the coronavirus (COVID-19) pandemic. This includes 4.5 million <a class="govuk-link govuk-link--no-visited-state" href="https://www.nhs.uk/conditions/coronavirus-covid-19/coronavirus-vaccination/book-coronavirus-vaccination/">vaccination</a> invitation letters and 700 million <a class="govuk-link govuk-link--no-visited-state" href="https://www.gov.uk/get-coronavirus-test">test results</a>.</p>
+  <p class="govuk-body">Since January 2021 we’ve sent X billion messages for coronavirus (COVID-19) services. This includes X million <a class="govuk-link govuk-link--no-visited-state" href="https://www.nhs.uk/conditions/coronavirus-covid-19/coronavirus-vaccination/coronavirus-booster-vaccine/">booster vaccination</a> invitations and X million <a class="govuk-link govuk-link--no-visited-state" href="https://www.gov.uk/get-coronavirus-test">test results</a>.</p>
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:58]:
```suggestion
  <p class="govuk-body">In 2021 we sent 1.3 billion messages for coronavirus (COVID-19) services. This includes 270 million <a class="govuk-link govuk-link--no-visited-state" href="https://www.nhs.uk/conditions/coronavirus-covid-19/coronavirus-vaccination/coronavirus-booster-vaccine/">booster vaccination</a> invitations and 751 million <a class="govuk-link govuk-link--no-visited-state" href="https://www.gov.uk/get-coronavirus-test">test results</a>.</p>
```

File: app/templates/views/roadmap.html [hunk 5]
```diff
@@ -27,34 +27,47 @@ <h2 class="heading-medium">Things we’re working on</h2>
...
+    <li>Explore ways to improve our daily letter printing capability</li>
 
   </ul>
 
   <h2 class="heading-medium" id="things-we-have-done">Things we’ve done</h2>
 
-  <p class="govuk-body">Since January 2021 we’ve sent 1 billion messages for services that were set up in response to the coronavirus (COVID-19) pandemic. This includes 4.5 million <a class="govuk-link govuk-link--no-visited-state" href="https://www.nhs.uk/conditions/coronavirus-covid-19/coronavirus-vaccination/book-coronavirus-vaccination/">vaccination</a> invitation letters and 700 million <a class="govuk-link govuk-link--no-visited-state" href="https://www.gov.uk/get-coronavirus-test">test results</a>.</p>
+  <p class="govuk-body">In 2021 we sent 1.3 billion messages for coronavirus (COVID-19) services. This includes 270 million <a class="govuk-link govuk-link--no-visited-state" href="https://www.nhs.uk/conditions/coronavirus-covid-19/coronavirus-vaccination/coronavirus-booster-vaccine/">booster vaccination</a> invitations and 751 million <a class="govuk-link govuk-link--no-visited-state" href="https://www.gov.uk/get-coronavirus-test">test results</a>.</p>
+
+  <p class="govuk-body">More recently we have:</p>
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:60]:
```suggestion
  <p class="govuk-body">We have also:</p>
```

File: app/templates/views/roadmap.html [hunk 6]
```diff
@@ -27,34 +27,47 @@ <h2 class="heading-medium">Things we’re working on</h2>
...
+    <li>Explore ways to improve our daily letter printing capability</li>
 
   </ul>
 
   <h2 class="heading-medium" id="things-we-have-done">Things we’ve done</h2>
 
-  <p class="govuk-body">Since January 2021 we’ve sent 1 billion messages for services that were set up in response to the coronavirus (COVID-19) pandemic. This includes 4.5 million <a class="govuk-link govuk-link--no-visited-state" href="https://www.nhs.uk/conditions/coronavirus-covid-19/coronavirus-vaccination/book-coronavirus-vaccination/">vaccination</a> invitation letters and 700 million <a class="govuk-link govuk-link--no-visited-state" href="https://www.gov.uk/get-coronavirus-test">test results</a>.</p>
+  <p class="govuk-body">In 2021 we sent 1.3 billion messages for coronavirus (COVID-19) services. This includes 270 million <a class="govuk-link govuk-link--no-visited-state" href="https://www.nhs.uk/conditions/coronavirus-covid-19/coronavirus-vaccination/coronavirus-booster-vaccine/">booster vaccination</a> invitations and 751 million <a class="govuk-link govuk-link--no-visited-state" href="https://www.gov.uk/get-coronavirus-test">test results</a>.</p>
+
+  <p class="govuk-body">We have also:</p>
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:60]:
```suggestion
  <p class="govuk-body">Recently we have:</p>
```

File: app/templates/views/roadmap.html [hunk 7]
```diff
@@ -27,34 +27,47 @@ <h2 class="heading-medium">Things we’re working on</h2>
...
+  <p class="govuk-body">In 2021 we sent 1.3 billion messages for coronavirus (COVID-19) services. This includes 270 million <a class="govuk-link govuk-link--no-visited-state" href="https://www.nhs.uk/conditions/coronavirus-covid-19/coronavirus-vaccination/coronavirus-booster-vaccine/">booster vaccination</a> invitations and 751 million <a class="govuk-link govuk-link--no-visited-state" href="https://www.gov.uk/get-coronavirus-test">test results</a>.</p>
+
+  <p class="govuk-body">Recently we have:</p>
+
+  <ul class="list list-bullet">
+
+    <li>improved our performance by upgrading the software we use</li>
+    <li>increased our capacity at short notice to meet a spike in demand</li>
+    <li>made quarterly billing reports available for organisations to download</li>
+
```
Comment (karlchillmaid) [inline app/templates/views/roadmap.html:67]:
```suggestion
    <li>made quarterly billing reports available for organisations to download</li>
```

**Review by benthorner — APPROVED** [review]
Reads and renders OK locally ✅

**Pull request merged**